staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -165,6 +170,9 @@ type DelegationTarget interface {
 
 	// ListedPaths returns the paths for supporting an index
 	ListedPaths() []string
+
+	// OpenAPISpec returns the OpenAPI spec of the delegation target if exists, nil otherwise.
+	OpenAPISpec() *spec.Swagger
```

Comment:
Do you mean I can call http serve method of delegate to get `/swagger.spec` without credentials?